### PR TITLE
Fixes master lint and makes hilberthotel.dm use tabs instead of spaces

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Real question is why travis didnt pick this up
image

## Why It's Good For The Game

Map template datums still sleep in new but thats a issue for later

## Changelog
:cl:
code: Fixes master lint and makes hilberthotel.dm use tabs instead of spaces
/:cl:

### CODE REVIEW

**Comment by TiviPlus** [comment]
Yes its tested hilberts works fine

**Comment by SpaceManiac** [comment]
In the future, it would be nice if the "rewrite the whole file" and "change a few lines" steps were in separate commits so they could be reviewed more easily.

**Comment by SpaceManiac** [comment]
misclick

**Review by SpaceManiac — APPROVED** [review]

File: code/modules/ruins/spaceruin_code/hilbertshotel.dm [hunk 1]
```diff
@@ -1,196 +1,196 @@
...
+	if(storedRooms["[roomNumber]"])
+		var/datum/turf_reservation/roomReservation = SSmapping.RequestBlockReservation(hotelRoomTemp.width, hotelRoomTemp.height)
+		hotelRoomTempEmpty.load(locate(roomReservation.bottom_left_coords[1], roomReservation.bottom_left_coords[2], roomReservation.bottom_left_coords[3]))
+		var/turfNumber = 1
+		for(var/i=0, i<hotelRoomTemp.width, i++)
+			for(var/j=0, j<hotelRoomTemp.height, j++)
+				for(var/atom/movable/A in storedRooms["[roomNumber]"][turfNumber])
+					if(istype(A.loc, /obj/item/abstracthotelstorage))//Don't want to recall something thats been moved
+						A.forceMove(locate(roomReservation.bottom_left_coords[1] + i, roomReservation.bottom_left_coords[2] + j, roomReservation.bottom_left_coords[3]))
+					turfNumber++
```
Comment (Dennok) [inline code/modules/ruins/spaceruin_code/hilbertshotel.dm:94]: Great! This silently break Hilbert's Hotel loading!

**Pull request merged**